src/styleguide/_patterns/05-pages/01-careers.json

```diff
@@ -19,6 +19,7 @@
       }
     },
     {
+      "url": "../../patterns/05-pages-02-locations/05-pages-02-locations.rendered.html",
```

Comment:
I think you can just use `link.pages-locations` and that should be rendered for you